packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts

```diff
@@ -518,13 +518,31 @@ export async function transformGraphQLSchema(context, options) {
   context.print.success(`GraphQL schema compiled successfully.\n\nEdit your schema at ${schemaFilePath} or \
 place .graphql files in a directory at ${schemaDirPath}`);
 
+  if (isAuthModeUpdated(options)) {
+    parameters.AuthModeLastUpdated = new Date();
+  }
   if (!options.dryRun) {
     JSONUtilities.writeJson(parametersFilePath, parameters);
   }
 
   return transformerOutput;
 }
 
+function isAuthModeUpdated(options): boolean {
```

Comment:
we should define this function in one place and reuse it

Reply:
updated